certificates/views.py

```diff
@@ -52,3 +66,37 @@ def get_context_data(self, **kwargs):
         context['certificate'] = certificate
 
         return context
+
+
+class ProgramCertificateView(CertificateView):
+    """
+    Views for program certificates
+    """
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
```

Comment:
Can you write tests covering these cases? You can access the context of the view and assert its values using the test client